h/static/scripts/form-select-onfocus-controller.js

```diff
@@ -0,0 +1,17 @@
+function FormSelectOnFocusController(element) {
+  var self = this;
+
+  this._formInputs = element.querySelectorAll('form input.js-select-onfocus');
```

Comment:
Selectors should be minimally specific, so just `.js-select-onfocus` would be fine here.